modules/core/server/views/layout.server.view.html

```diff
@@ -38,12 +31,13 @@
 	<!-- Fav Icon -->
 	<link href="/modules/core/img/brand/favicon.ico" rel="shortcut icon" type="image/x-icon">
 
-	<!--Application CSS Files-->
+	<!-- Application CSS Files -->
 	{% for cssFile in cssFiles %}<link rel="stylesheet" href="{{cssFile}}">{% endfor %}
 
-	<!-- HTML5 Shim -->
+	<!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
 	<!--[if lt IE 9]>
-		<script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
+		<script src="//oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+		<script src="//oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
```

Comment:
if we're going all the way with removing html5mode support as @rhutchison mentions then I'm ok with cleaning this up as well.
Question is whether removing this will completely make the framework unusable on IE9 or not?